SYS_ATL/new_eff.py

```diff
@@ -669,7 +665,7 @@ def is_empty(ls):
 
 
 E = ADT("""
-module Effects {
+module EffectsNew {
```

Comment:
The module name determines the entry in `sys.modules` so name clashes are incorrect. Detecting this error is TODO in asdl-adt.